source/upgrade/installing-license-key.rst

```diff
@@ -0,0 +1,63 @@
+Installing a License Key
+========================
+
+|enterprise| |professional| |self-hosted|
+
+.. |enterprise| image:: ../images/enterprise-badge.png
+  :scale: 30
+  :target: https://mattermost.com/pricing
+  :alt: Available in the Mattermost Enterprise subscription plan.
+
+.. |professional| image:: ../images/professional-badge.png
+  :scale: 30
+  :target: https://mattermost.com/pricing
+  :alt: Available in the Mattermost Professional subscription plan.
+
+.. |self-hosted| image:: ../images/self-hosted-badge.png
+  :scale: 30
+  :target: https://mattermost.com/deploy
+  :alt: Available for Mattermost Self-Hosted deployments.
+
+You can use the System Console, the mmctl, or the CLI to add or change a Mattermost license key.
+
+.. tabs::
+
+    .. tab:: Using System Console
+
+        1. Go go **System Console > About > Edition and License**.
+        2. Select **Remove Enterprise License and Downgrade Server**. This clears the license from the server and refreshes the System Console.
+        3. Upload the new license key file.
+
+        Once the key is uploaded and installed, the details of your license are displayed.
+
+    .. tab:: Using mmctl
+
+        Use the `mmctl license upload <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-license-upload>`__ command to to upload a new license or replace an existing license file with a new one. When complete, restart the Mattermost server. If you're running in a `High Availability <https://docs.mattermost.com/scale/high-availability-cluster.html>`__ environment, the new license file must be updated to every node.
+
+        .. code-block:: none
+
+            mmctl license upload [license] [flags]
+
+    .. tab:: Using the CLI
```

Comment:
Do we want to mention the legacy CLI is only available for versions v5.39 or earlier?